templates/nrfx_glue.h

```diff
@@ -98,6 +98,19 @@ extern "C" {
  */
 #define NRFX_IRQ_DISABLE(irq_number)
 
+/**
+ * @brief Macro for setting a specific IRQ as pending.
+ *
+ * @param irq_number  IRQ number.
+ */
+#define NRFX_IRQ_PENDING_SET(irq_number)
+
+/**
+ * @brief Macro for clearing a specific IRQ from pending.
```

Comment:
Does this sound ok? clear from pending?

Reply:
What about "clearing the pending status of a specific IRQ"?

Reply:
Hence, aligning the 'set' also to: "setting the pending IRQ status of a specific IRQ"?

Reply:
Not necessarily. The description for `_SET` looks fine as it is now.

Reply:
done.

Reply:
`NRFX_IRQ_PENDING_CLEAR` is not used in nrfx. Do you need it for a particular reason?
To be consistent, we should either not define this macro at all or define `NRFX_IRQ_IS_PENDING` as well to have a full set (but kind of forcing integrators to provide an implementation for unused macros).
I think I'd take this second path. @kl-cruz What you think?

Reply:
I can remove the CLEAR, or add the IS_SET. I'm fine with both. In my use case i'm only using _SET.